### PR TITLE
docs: grammar fixes for README.md, features, and known issues

### PR DESCRIPTION
**Pull request by davis274**
Todd, am I doing this right?

### CODE REVIEW

File: lib/spack/docs/features.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-================
+ ================
```
Comment (adamjstewart) [inline lib/spack/docs/features.rst]: I don't think this space should be here.

File: lib/spack/docs/known_issues.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ This is a list of known bugs in Spack. It provides ways of getting around these
 problems if you encounter them.
 
 -----------------------------------------------------------------
-Default variants are not taken into account during concretization
+Default variants are not taken into account during concretization.
```
Comment (adamjstewart) [inline lib/spack/docs/known_issues.rst]: None of the other section titles end in a period. I would leave this one off.

**Comment by davis274** [comment]
Thanks for catching this, Adam. Holly and I were trying to figure out the pull system and I made a quick change for use as an example. I hope not to bother you mavens further!
md

**Comment by adamjstewart** [comment]
@davis274 No worries, welcome to Spack! If you're new to Git, I would recommend the [Git Atlassian Tutorial](https://www.atlassian.com/git/tutorials) for a short tutorial or the [Pro Git Book](https://git-scm.com/book/en/v2) for a longer tutorial.

**Comment by tgamblin** [comment]
@davis274 @hauten: Thanks for the PR!

I missed this because you referred to "Todd" instead of @tgamblin (my email filters will catch the latter) -- can you use @tgamblin from now on?

The changes look mostly good to me -- I think my only request would be to leave the period off as @adamjstewart requested.  You should be able to push an update if you change this locally and commit.

File: lib/spack/docs/features.rst (hunk 1 above)
Comment (tgamblin) [reply lib/spack/docs/features.rst]: This space is causing the documentation tests to fail because Sphinx doesn't like it.

**Comment by tgamblin** [comment]
@davis274 @hauten: any updates on this?

**Comment by adamjstewart** [comment]
Go ahead and push.

**Comment by adamjstewart** [comment]
Can you resolve the merge conflicts?

**Comment by alalazo** [comment]
Any chance somebody is willing to revive this PR?

**Comment by alalazo** [comment]
@tgamblin @adamjstewart Do we want to port some of the modifications to the tutorial repo and rebase this one ourselves?

**Comment by alalazo** [comment]
Closing but feel free to reopen if this is still considered useful.

**Pull request closed without merging**